snuba/consumers/consumer.py

```diff
@@ -573,6 +574,8 @@ def process_message(
                         _LAST_INVALID_MESSAGE[snuba_logical_topic.name] = start
                         sentry_sdk.set_tag("invalid_message_schema", "true")
                         logger.warning(err, exc_info=True)
+                    if enforce_schema:
+                        raise
```

Comment:
What will this raise?

Reply:
It will reraise the ValidationError https://github.com/getsentry/sentry-kafka-schemas/blob/b1385c004934060cb329a9e47a5234dbd53e05b9/python/sentry_kafka_schemas/codecs/__init__.py#L36C1-L41